### PR TITLE
fix(#3569): prohibit numbers in attribute-name in XSD

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #3569

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@yegor256 please check

File: eo-maven-plugin/src/main/resources/org/eolang/maven/phi/to-phi.xsl [hunk 1]
```diff
@@ -27,9 +27,7 @@ SOFTWARE.
   <xsl:output encoding="UTF-8" method="text"/>
   <!-- Variables -->
   <xsl:variable name="aliases" select="program/metas/meta/part[last()]"/>
-  <xsl:variable name="number-pattern">
-    <xsl:value-of select="'^[0-9][1-9]*$'"/>
-  </xsl:variable>
+  <xsl:variable name="number-pattern" select="'^[0-9][1-9]*$'"/>
```
Comment (yegor256) [inline eo-maven-plugin/src/main/resources/org/eolang/maven/phi/to-phi.xsl:30]: @maxonfjvipon I believe, `10` won't match this pattern. Is it intended to be so?

File: eo-parser/src/main/resources/XMIR.xsd [hunk 2]
```diff
@@ -83,7 +83,7 @@ SOFTWARE.
       </xs:documentation>
     </xs:annotation>
     <xs:restriction base="xs:string">
-      <xs:pattern value="[0-9]+|@|[a-z]\S*"/>
+      <xs:pattern value="α[0-9]+|@|[a-z][^ \n\t\r]*"/>
```
Comment (yegor256) [inline eo-parser/src/main/resources/XMIR.xsd:86]: @maxonfjvipon here, `\S` means anything except space, maybe it's better to keep it?

File: eo-maven-plugin/src/main/resources/org/eolang/maven/phi/to-phi.xsl (hunk 1 above)
Comment (maxonfjvipon) [reply eo-maven-plugin/src/main/resources/org/eolang/maven/phi/to-phi.xsl:30]: @yegor256 no, it's a mistake

**Comment by yegor256** [comment]
@maxonfjvipon thanks!

**Comment by 0crat** [comment]
@maxonfjvipon Great work on your contribution! 🎉 You've earned +4 points, despite some areas for improvement. We noticed 201 hits-of-code (slightly over our 200 limit), no code review, and only 5 comments. However, we value your effort and added a bonus to encourage you. Your new balance is [+49](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep the contributions coming and remember to [focus on quality](https://www.yegor256.com/2018/03/06/speed-vs-quality.html)!

**Pull request merged**